unit-tests/src/main/scala/java/io/FileInputStreamSuite.scala

```diff
@@ -62,4 +62,10 @@ object FileInputStreamSuite extends tests.Suite {
     assert(fis.read() == 0xFF)
     assert(fis.read() == -1)
   }
+
+  testFails("throws FileNotFoundException already on creating new FileInputStream", 765) {
```

Comment:
Could you change the name of the test to say that it should throw if the file doesn't exist? Currently it's not very clear why it should throw an exception by reading the name of the test.